scylla/src/transport/session_test.rs

```diff
@@ -2798,3 +2798,98 @@ async fn simple_strategy_test() {
 
     assert_eq!(rows, vec![(1, 2, 3), (4, 5, 6), (7, 8, 9)]);
 }
+
+#[tokio::test]
+async fn manual_primary_key_computation() {
```

Comment:
Nit: add `_test` at the end, or `test_` at the beginning (I prefer the latter). Let's have a rule to name tests this way; it will make them stand out more and will reduce the risk of name collision with non-test functions.